testsuite/features/build_validation/core/srv_disable_default_schedule_tasks.feature

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Remove schedule tasks to manually trigger them when we consider the best moment
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Delete scheduled reposyncs
+    When I follow the left menu "Admin > Task Schedules"
+    And I follow "mgr-sync-refresh-default"
+    And I choose "disabled"
+    And I click on "Update Schedule"
+    And I click on "Delete Schedule"
```

Comment:
Thinking it twice, maybe deleting it is a too sharp approach. What about changing the cron, so instead of running every minute, it runs... every week?
That way, once our BV pipeline finish, we can consider a scenario to have the default cron values back here.